docker-compose.yml

```diff
@@ -0,0 +1,65 @@
+version: '3.7'
+
+services:
+  app: &app
+    build:
+      context: .
+      dockerfile: ./Dockerfile
+      args:
+        RUBY_VERSION: '2.3.1'
+        BUNDLER_VERSION: '1.16.6'
+        NODE_MAJOR: '10'
+    image: pivorak-web-app:0.0.1
+    tmpfs:
+      - /tmp
+
+  backend: &backend
+    <<: *app
+    stdin_open: true
+    tty: true
+    volumes:
+      - .:/app:cached
+      - rails_cache:/app/tmp/cache
+      - bundle:/bundle
+    environment:
+      - REDIS_URL=redis://redis:6379
+      - DATABASE_URL=postgres://postgres:postgres@postgres:5432
+    depends_on:
+      - postgres
+      - redis
+
+  runner:
+    <<: *backend
+    command: /bin/bash
+    ports:
+      - '3000:3000'
+
+  rails:
+    <<: *backend
+    command: bundle exec rails server -b 0.0.0.0
+    ports:
+      - '3000:3000'
+
+  sidekiq:
+    <<: *backend
+    command: bundle exec sidekiq -C config/sidekiq.yml
+
+  postgres:
+    image: postgres:9.5
+    volumes:
+      - postgres:/var/lib/postgresql/data
+    ports:
+      - 5432
+
+  redis:
+    image: redis:3.2-alpine
```

Comment:
Can anyone confirm that we are using Redis `3.2` in production?

Reply:
redis 2.8.4

Reply:
Seem like there is no simple way to get official Redis image of version 2.8.4 https://travis-ci.org/pivorakmeetup/pivorak-web-app/builds/580163032#L1532, let's upgrade it in production since test suite run is ok with redis 3.2 same with Postgers(README file tells "Note! You should have PostgreSQL >= 9.4" probably there are some reasons why)